src/test/java/ibm/jceplus/junit/openjceplusfips/TestRSA.java

```diff
@@ -49,34 +56,19 @@ public static Test suite() {
         return suite;
     }
 
-    // --------------------------------------------------------------------------
-    // This method is to check whether an algorithm is valid for the cipher
-    // but not supported by a given provider.
+        // --------------------------------------------------------------------------
+    // This method is to check whether OpenJCEPlusFIPS can throw exception with RSA cipher 
     //
-    @Override
-    public boolean isAlgorithmValidButUnsupported(String algorithm) {
-        if (algorithm.equalsIgnoreCase("RSAwithNoPad") || algorithm.equalsIgnoreCase("RSAforSSL")) {
```

Comment:
Maybe this is a pointer to the signatures that use RSA and should change as well?

Reply:
RSA signature issue will be addressed in another PR.

Reply:
Hello think we could describe at a high level what we plan on doing with this other update?

Reply:
Further investigation is certainly warranted for these two Signatures / Ciphers given the way in which the OCKC library is called using encipher and decipher. We will attempt to tackle that in a subsequent PR.